crates/solidity/testing/snapshots/bindings_output/built_ins/functions/generated/0.8.4-failure.txt

```diff
@@ -0,0 +1,166 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
```

Comment:
I'm not sure I understand why this is a `-failure.txt`? I don't see anything unresolved here.